examples/plotting/solar_cycle_example.py

```diff
@@ -1,33 +1,53 @@
 """
-==============================
-Plotting the solar cycle index
-==============================
+=============================
+Plotting a solar cycle index
+=============================
 
-How to plot the current and possible next solar cycle.
+This example demonstrates how to plot the solar cycle in terms of
+the number of sunspots and a prediction for the next few years.
 """
 import matplotlib.pyplot as plt
 
+from astropy.time import Time, TimeDelta
+import astropy.units as u
+
 import sunpy.timeseries as ts
-from sunpy.data.sample import NOAAINDICES_TIMESERIES, NOAAPREDICT_TIMESERIES
+from sunpy.net import Fido, attrs as a
+from sunpy.time import TimeRange
+
+###############################################################################
+# The U.S. Dept. of Commerce, NOAA, Space Weather Prediction Center (SWPC)
+# provides recent solar cycle indices which includes different sunspot numbers,
+# radio flux, and geomagnetic index. They also provide predictions for how the
+# sunspot number and radio flux will evolve. Predicted values are based on the
+# consensus of the Solar Cycle 24 Prediction Panel.
+#
+# We will first search for and then download the data.
+time_range = TimeRange("2008-06-01 00:00", Time.now())
+result = Fido.search(a.Time(time_range.start, time_range.end), a.Instrument('noaa-indices'))
```

Comment:
`
```suggestion
result = Fido.search(a.Time(time_range), a.Instrument('noaa-indices'))
```

Reply:
or was this .start and .end added for a reason?

Reply:
Do we need a time range for the search?

Reply:
Yes I think you need to include a time for a Fido search no?

Reply:
I thought that was removed. Maybe I'm wrong.